migrations/20170712152806_add_tos_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddTosTable extends AbstractMigration
+{
+    /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+        $this->table('tos')
+          ->addColumn('user_id', 'integer', ['null' => false])
+          ->addColumn('agreement_date', 'integer', ['null' => false])
```

Comment:
I'd usually suggest these should be mysql datetime column, not just integers. That said... if its working as-is we should really get this deployed. This part of the codebase is unlikely to change often so not a major